public-api/config/cdr_versions_prod.json

```diff
@@ -27,10 +27,10 @@
     "name": "All of Us Dataset v2",
     "dataAccessLevel": 1,
     "bigqueryProject": "aou-res-curation-prod",
-    "bigqueryDataset": "combined20191004_dbrowser_1",
+    "bigqueryDataset": "combined20191004_dbrowser_2",
     "creationTime": "2020-02-11 16:27:20Z",
     "releaseNumber": 2,
     "numParticipants": 225360,
```

Comment:
Should you update the participant count since participants have been removed?

Reply:
thanks for catching that. I have updated value